daft/ga.py

```diff
@@ -145,3 +147,41 @@ def run(self, **kwargs):
 
         # return the top chromosomes.
         return tools.selBest(population, k=kwargs.pop("k", 1))
+
+
+def combined_mutation(individual, low, up, indpb):
+    """
+    Firstly the type of mutation is choosed with 90% and 10% accordigly.
+    The first mutation type mutates an individual by replacing attributes,
+    with probability *indpb*, by a integer uniformly drawn between *low*
+    and *up* inclusively.The second mutation type mutates a whole individual
+    by replacing all attibuttes by ingegeres uniformly drawn by the initial chain.
+
+    Parameters
+    ----------
+    individual : np.ndarray
+                 individual to be mutated.
```

Comment:
The explanations (e.g. `individual to be mutated.`) have to be exactly *4 spaces* further indented that the names (e.g. `individual : np.ndarray`), as in all the other docstrings. `black` will not fix this, since it cannot manipulate strings.